src/test/pmemblk_create/README

```diff
@@ -0,0 +1,11 @@
+Linux NVM Library
+
+This is src/test/pmemblk_create/README.
+
+This directory contains a unit test for pmemblk_create().
```

Comment:
General comment about these tests: they seem to only test if the entry point returns an unexpected error code.  But when the functions being tested return success, perhaps there are other checks the program could do to make sure the expected behavior actually happened (like checking for existence of the file, or maybe using some of the other entry points to do I/O for _check() on the pool).